static/templates/actions_popover_content.hbs

```diff
@@ -39,7 +47,7 @@
     <li>
         <a class="popover_toggle_collapse" data-message-id="{{message_id}}" tabindex="0">
             <i class="fa fa-minus" aria-hidden="true"></i>
-            {{t "Collapse" }} <span class="hotkey-hint">(–)</span>
+            {{t "Collapse message" }} <span class="hotkey-hint">(–)</span>
```

Comment:
This should also move in the first commit. Basically we can move the tooltip changes (and documentation changes) to the same commit which changes the popover menu text.